compose/rpa-em-ccd-orchestrator.yml

```diff
@@ -7,8 +7,8 @@ services:
     environment:
       - SERVER_PORT=8080
       - CALLBACK_DOMAIN=rpa-em-ccd-orchestrator
-      - IDAM_API_BASE_URI=http://rse-idam-simulator:5000
-      - OPEN_ID_API_BASE_URI=http://rse-idam-simulator:5000/o
+      - IDAM_API_BASE_URI=${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}
```

Comment:
do we have to change this from similator?

Reply:
IDAM_STUB_SERVICE_NAME is set to http://rse-idam-simulator:5000 in .env so this will still work (assuming you are using the simulator). This now makes it compatible no matter what idam flavour you decide to use.

Reply:
Ok nice one